README.md

```diff
@@ -110,7 +117,7 @@ additions [list]:
         filepath [str]: local path to an image to be added
         class [str]: for Image Classification, the name of the class for the entire image
         boxes [list]: for Object Detection, a list of bounding boxes
-            box [list]: a list of length 5 encoding the following information - [class, x, y, width, height], where x, y, width, and height are all normalized to the image dimensions 
+            box [list]: a list of length 5 encoding the following information - [class, x, y, width, height], where x, y, width, and height are all normalized to the image dimensions
```

Comment:
Just FYI - this README is super out of date. Any section on LDF needs to be rewritten